pinc/graph_data.inc

```diff
@@ -101,11 +100,14 @@ function user_logging_on($past, $preceding)
     ///////////////////////////////////////////////////
     //query db and put results into arrays
 
-    $result = mysqli_query(DPDatabase::get_connection(), "
-      SELECT DATE_FORMAT(FROM_UNIXTIME(time_stamp),'$date_format'), $column_name
+    $sql = sprintf("SELECT DATE_FORMAT(FROM_UNIXTIME(time_stamp),'%s'), %s
       FROM user_active_log
-      WHERE time_stamp >= $min_timestamp
-      ORDER BY time_stamp");
+      WHERE time_stamp >= %d
+      ORDER BY time_stamp",
+      DPDatabase::escape($date_format),
+      DPDatabase::escape($column_name),
+      $min_timestamp);
+    $result = DPDatabase::query($sql);
 
```

Comment:
Column spacing? Also, add space before the `%s` in `SELECT DATE_FORMAT(FROM_UNIXTIME(time_stamp),'%s')`?